src/main/java/com/demo/jwt/JwtMybatisApplication/controller/StudentController.java

```diff
@@ -6,6 +6,7 @@
 import com.demo.jwt.JwtMybatisApplication.dto.StudentsDisplayDto;
 import com.demo.jwt.JwtMybatisApplication.model.SubjectEntity;
 import com.demo.jwt.JwtMybatisApplication.service.StudentService;
+import jakarta.annotation.security.RolesAllowed;
```

Comment:
More authorization related validations and checks should be added.
Please think of those and add all of them.